sql/src/main/java/org/apache/druid/sql/calcite/aggregation/builtin/EarliestLatestAnySqlAggregator.java

```diff
@@ -262,6 +306,17 @@ public RelDataType inferReturnType(SqlOperatorBinding sqlOperatorBinding)
                   "'" + aggregatorType.name() + "(expr, maxBytesPerString)'\n",
                   OperandTypes.ANY,
                   OperandTypes.and(OperandTypes.NUMERIC, OperandTypes.LITERAL)
+              ),
+              OperandTypes.sequence(
+                  "'" + aggregatorType.name() + "(expr, timeColumn)'\n",
+                  OperandTypes.ANY,
+                  OperandTypes.ANY
```

Comment:
any seems a bit permissive, since i think this probably needs to be a long?

Reply:
true, moved it to `NUMERIC` - should we make it to `EXACT_NUMERIC` or does this look ok? long type isn't there. there's `INTEGER` too but I'm not sure if its in a mathematical sense or not.